### PR TITLE
[DBIO-1598] refactor: Refactor substrate module in indexer

### PR DESCRIPTION
**Pull request by rubenkristian**
JIRA LINK 🟦
- https://blocksphere2020.atlassian.net/browse/DBIO-1598

Changelog/Description 🗒️
- move event command to event folder in substrate folder module
- update path in unit test

### WARNING ⚠️
-

### CODE REVIEW

File: test/unit/events/genetic-testing/genetic-testing.command.spec.ts [hunk 1]
```diff
@@ -48,20 +51,13 @@ describe("Genetic Testing Substrate Event Handler", () => {
     const modules: TestingModule = await Test.createTestingModule({
 			imports: [
```
Comment (agustinustheo) [inline test/unit/events/genetic-testing/genetic-testing.command.spec.ts]: Please remove `imports` from tests.

File: test/unit/events/certifications/certifications.command.spec.ts [hunk 2]
```diff
@@ -66,20 +61,13 @@ describe("Certifications Substrate Event Handler", () => {
     const modules: TestingModule = await Test.createTestingModule({
 			imports: [
 				CommonModule,
-				ElasticsearchModule.registerAsync({
-					useFactory: async () => ({
-						node: process.env.ELASTICSEARCH_NODE,
-					}),
-				}),
 				CqrsModule,
 			],
```
Comment (agustinustheo) [inline test/unit/events/certifications/certifications.command.spec.ts]: Please remove `imports` from tests.

File: test/unit/events/labs/labs.handler.spec.ts [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+		});
+
+		it("Lab Updated Verification Status Handler", async () => {
+			const lab = createMockLab();
+			
+			const labUpdatedVerificationStatusCommand: LabUpdateVerificationStatusCommand = new LabUpdateVerificationStatusCommand([lab], mockBlockNumber());
+			await labUpdateVerificationStatusHandler.execute(labUpdatedVerificationStatusCommand);
+			expect(elasticsearchService.update).toHaveBeenCalled();
+		});
+	});
```
Comment (agustinustheo) [inline test/unit/events/labs/labs.handler.spec.ts]: The spacing here is pretty messed up, can you fix this?

File: test/unit/events/service-request/service-request.command.spec.ts [hunk 4]
```diff
@@ -110,20 +117,13 @@ describe("Service Request Substrate Event Handler", () => {
     const modules: TestingModule = await Test.createTestingModule({
 			imports: [
 				CommonModule,
-				ElasticsearchModule.registerAsync({
-					useFactory: async () => ({
-						node: process.env.ELASTICSEARCH_NODE,
-					}),
-				}),
 				CqrsModule,
 			],
```
Comment (agustinustheo) [inline test/unit/events/service-request/service-request.command.spec.ts]: Please remove `imports` from tests.

**Review by agustinustheo — DISMISSED** [review]

**Review by hilyds — APPROVED** [review]
LGTM

**Pull request merged**